### ISSUE TITLE
support comments

### ISSUE THREAD

**Issue by yorkie** [weflex/node-orgmode#6]
See https://github.com/weflex/node-orgmode/blob/master/org-syntax.org#comments, we should save comment lines at every block object.

**Comment by b232wang** [comment]
@yorkie  I think we should support `block` first. see https://github.com/weflex/node-orgmode/blob/master/org-syntax.org#greater-blocks. `comment lines` should inside of `block` or `heading`?

**Comment by yorkie** [comment]
You are right, Steven, created the new issue: https://github.com/weflex/node-orgmode/issues/8
